test/errhandling/lydia/rethrownGenerally-nested.chpl

```diff
@@ -0,0 +1,17 @@
+// Code from issue opened by BryantLam on Github
```

Comment:
I don't see any reason not to just put the issue # - e.g. code from issue #10224